app/utils/IncomeSourcesUtils.scala

```diff
@@ -71,49 +103,49 @@ object IncomeSourcesUtils {
 
   case class MissingKey(msg: String)
 
-  def getBusinessDetailsFromSession(implicit user: MtdItUser[_]): Either[Throwable, CheckBusinessDetailsViewModel] = {
+  def getBusinessDetailsFromSession(sessionService: SessionService)(implicit user: MtdItUser[_], ec: ExecutionContext): Future[CheckBusinessDetailsViewModel] = {
```

Comment:
Please return Future[Either[Throwable,CheckBusinessDetailsViewModel ]] instead

Reply:
Changed to Future[Either[Throwable,CheckBusinessDetailsViewModel ]]